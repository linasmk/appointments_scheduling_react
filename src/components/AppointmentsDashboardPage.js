/* ========= App Dependencies ============= */
import React from "react";

/* ========= Components ============= */
import AddAppointment from "./crud_components/AddAppointment";
import SearchAppointments from "./crud_components/SearchAppointments";
import ListAppointments from "./crud_components/ListAppointments";
/* ========= Code ============= */

const AppointmentsDashboardPage = () => (
  <div className="main">
    <article className="content-wrapper">
      <AddAppointment />
      <SearchAppointments />
      <ListAppointments />
    </article>
  </div>
);
export default AppointmentsDashboardPage;
