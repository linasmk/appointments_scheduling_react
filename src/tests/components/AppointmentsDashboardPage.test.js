import React from "react";
import { shallow } from "enzyme";
import AppointmentsDashboardPage from "../../components/AppointmentsDashboardPage";

test("Should AppointmentsDashboardPage render corectly", () => {
  const wrapper = shallow(<AppointmentsDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
