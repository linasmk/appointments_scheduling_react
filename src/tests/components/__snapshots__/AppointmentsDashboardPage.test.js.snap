// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should AppointmentsDashboardPage render corectly 1`] = `
<div
  className="main"
>
  <article
    className="content-wrapper"
  >
    <Connect(AddAppointment) />
    <Connect(SearchAppointments) />
    <Connect(ListAppointments) />
  </article>
</div>
`;
